documentation/asciidoc/topics/ref_cluster_admin.adoc

```diff
@@ -0,0 +1,21 @@
+Log into your {k8s} cluster as an administrator.
+
+ifndef::productized[]
+[source,options="nowrap"]
+----
+$ oc cluster up
+$ oc login -u system:admin
+----
+endif::productized[]
+
+ifdef::productized[]
+[source,options="nowrap"]
+----
+$ oc login -u system:admin
+----
+endif::productized[]
+
+[NOTE]
+====
+{okd} 3.11 is the minimum version for the {ispn_operator}.
```

Comment:
Since we are targeting Kubernetes in community should we have a K8s version reference here too?